games-strategy/freeciv/freeciv-2.5.6.recipe

```diff
@@ -0,0 +1,126 @@
+published by Sid Meier and Microprose [tm].  A few rules are different \
+where we think it makes more sense, and we have lots and lots of \
+adjustable parameters to make customizing games possible.
+Freeciv has been implemented completely independently of Civilization; \
+you do not need to own Civilization to play Freeciv."
+HOMEPAGE="http://www.freeciv.org"
+COPYRIGHT="2002-2015 The Freeciv Team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://prdownloads.sourceforge.net/freeciv/freeciv-$portVersion.tar.bz2"
+CHECKSUM_SHA256="cfadc23eec11a5a8ae74a7d3719eb126026137f0700da065367fd55cbab6999b"
+SOURCE_DIR="freeciv-2.5.6"
+ADDITIONAL_FILES="freeciv.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+	settings/freeciv/database.lua keep-old
+	"
+
+PROVIDES="
+	freeciv$secondaryArchSuffix = $portVersion
+	app:freeciv$secondaryArchSuffix = $portVersion
+	cmd:freeciv_manual$secondaryArchSuffix
+	cmd:freeciv_sdl$secondaryArchSuffix
+	cmd:freeciv_server$secondaryArchSuffix
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:liblzma$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libreadline$secondaryArchSuffix
+	lib:libsdl$secondaryArchSuffix
+	lib:libsdl_gfx$secondaryArchSuffix
+	lib:libsdl_image$secondaryArchSuffix
+	lib:libsdl_mixer$secondaryArchSuffix
+	lib:libsdl_ttf$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:liblzma$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libreadline$secondaryArchSuffix
+	devel:libsdl$secondaryArchSuffix
+	devel:libsdl_gfx$secondaryArchSuffix
+	devel:libsdl_image$secondaryArchSuffix
+	devel:libsdl_mixer$secondaryArchSuffix
+	devel:libsdl_ttf$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:autoconf
+	cmd:aclocal
+	cmd:libtool
+	cmd:gettext$secondaryArchSuffix
+	cmd:find
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	autoreconf -fi
+	runConfigure ./configure --datarootdir=$dataRootDir --datadir=$dataDir \
+		--enable-client=sdl --enable-fcmp=no
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	mkdir -p $appsDir/Freeciv
+	mv $binDir/freeciv-sdl $appsDir/Freeciv/Freeciv
+
```

Comment:
As there is no othe binary in the Freeciv folder maybe we could move it to $appsDir/Freeciv?

mkdir -p $appsDir
mv $binDir/freeciv-sdl $appsDir/Freeciv

this would also mean you'd have to adjust the other referals in the recipe to "$appsDir/Freeciv/Freeciv"